starter-code/src/functions-and-arrays.js

```diff
@@ -11,10 +24,47 @@ var words = [
   'crackpot'
 ];
 
+
+function findLongestWord(words) {
+  var longestWord = '';
+
+  for (i=0; i<words.length; i++) {
+    if (words[i].length > longestWord.length) {
+      longestWord = words[i];
+    }
+  }
+  return longestWord;    
+}
+
+words.find(findLongestWord);
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+
+function sumArray(p,n) {
+  for (j = 0; j < numbers.length; j++){
+    return p+n;
+  }
+}
+
+numbers.reduce(function sumArray(p,n){
+  return p + n;
+},0);
+
+
+var counter = 0;
+
+function sumArray(p,n) {
```

Comment:
This solution is valid, but you need to put the variable counter inside the function. The previous reduce has some problems with the parameters (we will talk about that later). Don't forget to delete the previous sumArray, you can't have two functions with the same name.